aequilibrae/paths/route_choice.py

```diff
@@ -104,8 +104,8 @@ def set_choice_set_generation(self, /, algorithm: str, **kwargs) -> None:
         excluded from the PSL calculations. The route is still returned, but with a probability of 0.0.
 
         The `cutoff_prob` should be in the range [0, 1]. It is then rescaled internally to [0.5, 1] as probabilities
```

Comment:
Maybe we should clarify the comment regarding re-scaling or move it to the implementation below. The bottom line is that we are using a binary logit model between the shortest path and each path in the choice set, which means that a probability of 0.5 corresponds to equal utility. The shortest path will always have higher utility than all produced paths so  will always have probability larger than 0.5 and therefore providing larger values would lead to exclusion of all alternatives. We just re-scale to make the cutoff probability parameter easier to interpret.